pinit.php

```diff
@@ -43,36 +43,56 @@ function pit_admin_scripts() {
 /*=== SHORTCODES
  *==============================*/
 
+function pit_follow_shortcode( $atts ) {
+	$atts = extract( shortcode_atts( array(
+		'url' => 'http://www.pinterest.com/pinterest/',
+		'text' => 'Follow',
+	), $atts ) );
+
+	return '<a data-pin-do="buttonFollow" href="' . $url . '">' . $text . '</a>';
+}
+
 function pit_pin_shortcode( $atts ) {
 	$atts = extract( shortcode_atts( array(
 		'url' => 'http://www.pinterest.com/pin/99360735500167749/',
+		'size' => 'small',
 	), $atts ) );
 
-	return '<a data-pin-do="embedPin" href="' . $url . '"></a>';
+	if ($size == 'large' || $size == 'medium') {
+		$width = 'data-pin-width="' . $size . '" ';
+	}
+
+	else {
+		$width = '';
+	}
```

Comment:
yep, good point. As I said [here](https://github.com/deshack/pinit/pull/15#discussion_r52878758) I just followed your codebase and made some copy/paste/mods but if you want improve it, I can only agree :)